src/ignitus-Shared/ignitus-DesignSystem/ignitus-Assets/ignitus-Icons/mediumIcon/mediumIcon.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {HTMLAttributes} from 'react';
+import {MediumIconColor} from '../../../ignitus-Atoms/colors';
+
+type Props = HTMLAttributes<SVGElement>;
+
+const MediumIcon = ({title = 'Medium Icon', ...rest}: Props) => {
+  return (
+    <svg
+      aria-label={title}
+      role="img"
+      viewBox="0 0 195 195"
+      xmlns="http://www.w3.org/2000/svg"
+      {...rest}
+    >
+      <title>{title}</title>
+      <rect
+        id="Rectangle-path"
+        fill={MediumIconColor}
```

Comment:
Remove fill from all the icons.

Reply:
- referencing colors give developers the ability to change color easily in the future and has no harm

Reply:
Fill is not required at all.